### PR TITLE
Adding missing pages

### PR DESCRIPTION
**Pull request by dorlinski**
[Problem] Thera are some missing pages for TAU 4.0/5.0 documentation
[Solution] Add three most importat pages: NumberPicker, DatePicer, TimePicker

Signed-off-by: Dominik Orliński <d.orlinski@samsung.com>

### CODE REVIEW

**Comment by mijin85cho** [comment]
It looks like new pages are added, but they are not listed up in toc/index/cover files.

Would you please check below files, to clarify the location of newly added files?

- **docs/application/web/api/toc.xml** 
   menu tree file for whole api reference files
- **docs/application/web/api/5.0/index.htm** 
   menu tree file for web api reference files.
- **docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_component_list.htm**
   cover page which list up sub pages under Wearable components

**Comment by dorlinski** [comment]
@TomaszLukawskiSam  please review

File: docs/application/web/api/toc.xml [hunk 1]
```diff
@@ -5767,6 +5770,9 @@
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm" label="ToggleSwitch"></topic>
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_viewSwitcher.htm" label="ViewSwitcher"></topic>
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_virtuallist.htm" label="VirtualList"></topic>
+						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_numberpicker.htm" label="NumberPicker"></topic>
```
Comment (mijin85cho) [inline docs/application/web/api/toc.xml]: It looks like these files are to be added to 4.0 and 3.0 also, according to this toc.xml update.
(Actually for 3.0, I can't sure as 3.0 is not commented in description.)
> [Problem] Thera are some missing pages for TAU 4.0/5.0 documentation

However as now the files are only added at 5.0,
would you please add `wearable_numberpicker.htm`, `wearable_timepicker.htm` and `wearable_datepicker.htm` files under desired folders and update their index.htm & wearable_component_list.htm file too?
Under 
   - /docs/application/web/api/3.0/ui_fw_api/Wearable_UIComponents/
   - /docs/application/web/api/4.0/ui_fw_api/Wearable_UIComponents/

Also need fix for
 - /docs/application/web/api/3.0/index.htm
 - /docs/application/web/api/3.0/ui_fw_api/Wearable_UIComponents/wearable_component_list.htm
 - /docs/application/web/api/4.0/index.htm
 - /docs/application/web/api/4.0/ui_fw_api/Wearable_UIComponents/wearable_component_list.htm

If we don't need this adding to 3.0, you don't need fix for 3.0.

Comment (dorlinski) [reply docs/application/web/api/toc.xml]: @lslachciak

Comment (lslachciak) [reply docs/application/web/api/toc.xml]: This change shouldn't be applied for 3.0

Comment (dorlinski) [reply docs/application/web/api/toc.xml]: I remove links to date picker, number picker and time picker in file docs/application/web/api/toc.xml for 3.0 and 4.0.

**Comment by mijin85cho** [comment]
Thanks for the update @dorlinski.
Also I found another missed issue later.

The wearable components are in alphabetical order now,
but Numberpicker, TimePicker, and DatePicker are not.

Would you please sort the additional topics in alphabetical order, 
if there is no reason for current order?

**Comment by dorlinski** [comment]
> Thanks for the update @dorlinski.
> Also I found another missed issue later.
> 
> The wearable components are in alphabetical order now,
> but Numberpicker, TimePicker, and DatePicker are not.
> 
> Would you please sort the additional topics in alphabetical order,
> if there is no reason for current order?

Done.

File: docs/application/web/api/toc.xml [hunk 2]
```diff
@@ -1982,6 +1982,9 @@
...
```
Comment (mijin85cho) [inline docs/application/web/api/toc.xml]: @dorlinski I think you missed this file to update.

Comment (dorlinski) [reply docs/application/web/api/toc.xml]: Fixed

**Pull request merged**